sdk/src/main/java/io/opentelemetry/sdk/OpenTelemetrySdk.java

```diff
@@ -40,7 +41,7 @@
    * @since 0.1.0
    */
   public static TracerSdkProvider getTracerProvider() {
-    return (TracerSdkProvider) OpenTelemetry.getTracerProvider();
+    return (TracerSdkProvider) ((Obfuscated<?>) OpenTelemetry.getTracerProvider()).unobfuscate();
```

Comment:
@bogdandrutu this is where the `Obfuscated` interface is used. `OpenTelemetrySdk.getTracerProvider()` needs some way to unwrap and retrieve the `TracerSdkProvider`. Did you have other idea how to unwrap?